packages/osquery_manager/kibana/osquery_saved_query/osquery_manager-b2c3d4e5-f6a7-8901-bcde-f12345678901.json

```
@@ -0,0 +1,201 @@
+        "ecs_mapping": [
+            {
+                "key": "event.category",
+                "value": {
+                    "value": ["configuration"]
+                }
+            },
+            {
+                "key": "event.type",
+                "value": {
+                    "value": ["info"]
+                }
+            },
+            {
+                "key": "event.kind",
+                "value": {
+                    "value": "state"
+                }
+            },
+            {
+                "key": "event.module",
+                "value": {
+                    "value": "osquery"
+                }
+            },
+            {
+                "key": "event.dataset",
+                "value": {
+                    "value": "osquery.crontab"
+                }
+            },
+            {
+                "key": "host.os.type",
+                "value": {
+                    "value": "linux"
+                }
+            },
+            {
+                "key": "rule.description",
+                "value": {
+                    "field": "detection_reason"
+                }
+            },
+            {
+                "key": "rule.name",
+                "value": {
+                    "field": "detection_method"
+                }
+            },
+            {
+                "key": "rule.category",
+                "value": {
+                    "field": "cron_type"
+                }
+            },
+            {
+                "key": "service.name",
+                "value": {
+                    "field": "crontab_owner"
+                }
+            },
+            {
+                "key": "process.command_line",
+                "value": {
+                    "field": "command"
+                }
+            },
+            {
+                "key": "process.executable",
+                "value": {
+                    "field": "executable_path"
+                }
+            },
+            {
+                "key": "file.path",
+                "value": {
+                    "field": "crontab_path"
+                }
+            },
```

This is unrelated to the other file details. Move to `event.provider`?
```suggestion
                "key": "event.provider",
                "value": {
                    "field": "crontab_path"
                }
            },
```